sql/src/main/java/io/crate/analyze/relations/RelationAnalyzer.java

```diff
@@ -343,6 +344,22 @@ protected AnalyzedRelation visitQuerySpecification(QuerySpecification node, Stat
         AnalyzedRelation relation;
         if (context.sources().size() == 1) {
             AnalyzedRelation source = Iterables.getOnlyElement(context.sources().values());
+
+            AliasedAnalyzedRelation aliasedRelation = null;
+
+            if (source instanceof AliasedAnalyzedRelation
```

Comment:
Is this the workaround we talked about? If so, please also add some comments that this is necessary for the LogicalPlanner to be able to do the `Get` optimization correctly.